wallets/src/test/java/network/misq/wallets/bitcoind/BitcoindSendTests.java

```diff
@@ -0,0 +1,169 @@
+package network.misq.wallets.bitcoind;
+
+import network.misq.wallets.AddressType;
+import network.misq.wallets.bitcoind.responses.ListTransactionsResponseEntry;
+import network.misq.wallets.bitcoind.responses.ListUnspentResponseEntry;
+import network.misq.wallets.rpc.RpcClient;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BitcoindSendTests {
```

Comment:
I get the second test failed as bitcoind shuts down in between.

Reply:
Did you had a running regtest instance before starting the tests? I do encounter the same situation if an instance was running before starting the tests. I do not want to kill a running instance. Who knows why the user has bitcoind running.

Reply:
No i did not had one running.